tests/fixtures-valid.js

```diff
@@ -27,22 +53,60 @@ const uniqueValues = {
 };
 
 const promises = [];
-
-// search fixture files
 const fixturePath = path.join(__dirname, `..`, `fixtures`);
-for (const manKey of fs.readdirSync(fixturePath)) {
-  const manDir = path.join(fixturePath, manKey);
-
-  // files in manufacturer directory
-  if (fs.statSync(manDir).isDirectory()) {
-    for (const file of fs.readdirSync(manDir)) {
-      if (path.extname(file) === `.json`) {
-        const fixKey = path.basename(file, `.json`);
-        handleFixtureFile(manKey, fixKey);
+
+
+if (args.fixtures !== undefined) {
+  // were assuming the fixtures are given as a semicolon
+  // separated list in the form `<manufacturer>[/[fixtures]]`
+  // where [fixtures] is a comma separated list of concrete fixtures
+
+  const fixturesToParse = args.fixtures;
+  const allManufacturers = fs.readdirSync(fixturePath);
+  for (const manufacturerFixtures of fixturesToParse.split(`;`)) {
+    let [manKey, fixtures] = manufacturerFixtures.split(`/`);
```

Comment:
```suggestion
    const [manKey, fixtures] = manufacturerFixtures.split(`/`);
```
We prefer `const` over `let` wherever possible, and as `manKey` and `fixtures` have a static value in this block, we could make them `const`.

Reply:
Right, I do as well but I had to use `let` because of some previous experiments and I just forgot to change it to `const` afterwards.